Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -56,8 +56,7 @@ class AppSettingsTableViewController: SettingsTableViewController {
         ]
 
         let accountChinaSyncSetting: [Setting]
-        let locale = NSLocale.currentLocale()
-        if locale.localeIdentifier != "zh_CN" {
+        if !(profile as! BrowserProfile).isChinaEdition {
```

Comment:
I'm not wild about this construction. Explicit casting may work, but will fail drastically if you try and test the app with a MockProfile. 

Suggest either:
 1. Move `isChinaEdition` into the Profile protocol, and provide a dummy implementation in `MockProfile` OR
 2. Explicitly test with a `if let` to check if it is safe to down cast to `BrowserProfile`.